packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,58 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { FASTSlider } from "../slider";
+import { SliderOrientation } from "../slider/slider";
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    @observable
+    public root: HTMLDivElement;
+
+    @attr
+    public max;
+
+    @attr
+    public position: string;
+
+    @attr
+    public label: string;
+
+    @attr({ attribute: "hide-mark", mode: "boolean" })
+    public hideMark: boolean = false;
+
+    private sliderMax: number;
+    private sliderMin: number;
+    private sliderOrientation: SliderOrientation;
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.sliderMax = (this.parentNode as FASTSlider).max;
+        this.sliderMin = (this.parentNode as FASTSlider).min;
```

Comment:
I think this is a clever implementation, but what happens if I use this with my own slider which is not FASTSlider?

Reply:
Maybe worth extracting an interface that defines the minimal properties needed to use the label with a slider. I think it's just min and max. Things like orientation and direction could be optional on that interface and this code could assume defaults if not present.

Reply:
I've fixed this to deal with there not being a parent fast-slider, but now I'll do the interface thing.

Reply:
interface in place